modules/vaos/spec/services/jwt_spec.rb

```diff
@@ -1,12 +1,13 @@
 # frozen_string_literal: true
 
 require 'rails_helper'
+require_relative '../support/fixture_helper'
 
 describe VAOS::JWT do
   subject { VAOS::JWT.new(user) }
 
   let(:user) { build(:user, :vaos) }
-  let(:rsa_private) { OpenSSL::PKey::RSA.generate(4096) }
+  let(:rsa_private) { OpenSSL::PKey::RSA.new(read_fixture_file('open_ssl_rsa_private.pem')) }
```

Comment:
Definitely better.

It's worth documenting somewhere how the file is generated (maybe just a comment line in the file itself?) since now its not necessarily clear that it was with the 4096 param.

Reply:
I'm not sure it needs to be anything specific per se, certainly not for specs. I say this because according to jwt.io you can have a lot of different types of algos, it's part of the header. Would think one could infer the algo from the header. Usually the first few bytes includes this information.

Reply:
I'm happy to add a comment if you feel it's necessary, but for example:

`openssl rsa -in test.key -text | head -n 1`                                                                                                                                       
Private-Key: (4096 bit)